### PR TITLE
fix: remove unecessary paddings / margings on home and team page

### LINKED ISSUE #2717: [Improvement] Paddings & Margins |  Team & Team Member Page
Check Fix unnecessary paddings In Team & Team Member Page

screenshot-localhost_3030-2024_07_13-00_26_09

screenshot-localhost_3030-2024_07_13-00_17_02

### PR DESCRIPTION
**Pull request by CREDO23**
### This PR fixes #2717 

Screenshots :
Screenshot from 2024-07-18 13-22-44
Screenshot from 2024-07-18 13-22-22

### CODE REVIEW

File: apps/web/app/[locale]/page-component.tsx [hunk 1]
```diff
@@ -65,69 +65,56 @@ function MainPage() {
...
-								>
-									<div>{isTeamMember ? <TeamMembers kanbanView={view} /> : <NoTeam />}</div>
-								</ResizablePanel>
-							</ResizablePanelGroup>
-						</div>
-					</MainLayout>
-				</div>
-				<div className="bg-white dark:bg-transparent ">
-					<Divider />
-					<Footer
```
Comment (evereq) [inline apps/web/app/[locale]/page-component.tsx:127]: @CREDO23 footer removed, why!?

Comment (CREDO23) [reply apps/web/app/[locale]/page-component.tsx:127]: It was a duplicate !
There is a Footer in the `<MainLayout/>` component.

**Review by Cedric921 — CHANGES_REQUESTED** [review]
Now we have unecessary spacing in other places, Please check and Fix @CREDO23 
review2

review1

**Comment by CREDO23** [comment]
done @Cedric921

**Review by Cedric921 — APPROVED** [review]

**Pull request merged**